packages/xo-web/src/xo-app/home/pool-item.js

```diff
@@ -87,9 +87,14 @@ export default class PoolItem extends Component {
     return icon(tooltip)
   }
 
+  _isXcpngPool() {
+    return Object.values(this.props.poolHosts)[0].productBrand === 'XCP-ng'
```

Comment:
Should we verify if at least one host is an XCP-ng? 
```suggestion
    return Object.values(this.props.poolHosts).some(host => host.productBrand === 'XCP-ng')
```

Reply:
It seems to me that hosts within a pool are always on the same `productBrand / version`.

cc/ @julien-f

Reply:
I think that's fair to assume indeed.

(The version may probably be different during rolling pool upgrade but that's an edge case.)

Reply:
Indeed, if it happens it's only a transient state, so it's an acceptable assumption.